packages/components/src/entities/SampleCreatePage.spec.tsx

```diff
@@ -90,7 +90,7 @@ describe('SampleCreatePage', () => {
         );
         await waitForLifecycle(wrapper);
         validate(wrapper);
-        expect(wrapper.find(EntityInsertPanel).prop('canEditEntityTypeDetails')).toBeTruthy();
+        expect(wrapper.find(EntityInsertPanel).prop('canEditEntityTypeDetails')).toBeFalsy();
```

Comment:
was this `canEditEntityTypeDetails` prop removed? if so, is this falsy because it is undefined? should this check just be removed in that case?

Reply:
The property was not removed from EntityInsertPanel, no.